tracer/test/Datadog.Trace.Tests/Telemetry/config_norm_rules.json

```diff
@@ -473,6 +473,8 @@
   "DD_IAST_REDACTION_ENABLED": "iast_redaction_enabled",
   "DD_IAST_REDACTION_KEYS_REGEXP": "iast_redaction_keys_regexp",
   "DD_IAST_REDACTION_VALUES_REGEXP": "iast_redaction_values_regexp",
+  "DD_IAST_REDACTION_NAME_PATTERN": "iast_redaction_name_pattern",
+  "DD_IAST_REDACTION_VALUE_PATTERN": "iast_redaction_value_pattern",
```

Comment:
Don't forget the PR to dd-go to update the values there too 🙂

Reply:
Here is the [PR](https://github.com/DataDog/dd-go/pull/124367)